tests/jeos/firstrun.pm

```diff
@@ -50,6 +50,16 @@ sub verify_user_info {
     # User has locale defined in firstboot, root always defaults to POSIX (i.e. English)
     my $proglang = $args{user_is_root} ? 'en_US' : $lang;
     assert_script_run("ldd --help | grep '^" . $lang_data{$proglang} . "'");
+
+    return if $user;
+
+    if (script_run('rpm -q system-group-wheel') == 0) {
+        if (script_run('getent group wheel') && is_sle('16+')) {
```

Comment:
`getent group wheel` only verifies existence of the group, not membership

Reply:
changed